packages/pwa/app/utils/utils.js

```diff
@@ -154,3 +158,85 @@ export const capitalize = (text) => {
         .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
         .join(' ')
 }
+
+export const getAppConfig = () => {
+    return pwaKitConfig.app
+}
+
+/**
+ * get siteId based on the site configuration
+ * @param appConfig
+ * @param originalUrl
+ * @param appOrigin
+ * @returns {string|undefined|*}
+ */
+export const getSiteId = (appConfig = {}, originalUrl, appOrigin) => {
```

Comment:
we've been discussing having a single place for configurations and possible we could pass the configuration to the SDK, then we could move this function to the SDK, so we could simply the method signature a bit.

Ideally i think we could simplify the parameters to only pass in a url.

```suggestion
export const getSiteId = (url) => {
```

Reply:
This function deserves a really good documentation, because it is complicated and opinionated.

Reply:
For the parameter names, I think it's better if we stick with names that are found in the standard [Location API](https://developer.mozilla.org/en-US/docs/Web/API/Location) for better readability. 

```suggestion
export const getSiteId = (appConfig = {}, pathname, origin) => {
```